bundles/specmate-persistency-cdo/src/com/specmate/persistency/cdo/internal/CDOPersistencyServiceConfig.java

```diff
@@ -96,7 +97,7 @@ private void startMonitoringThread() {
 					}
 				}
 			} else {
-				if (checkConnection()) {

```

Comment:
Why removing this check makes sense? Looks like you are changing the behaviour of this method